.github/workflows/linting.yaml

```
@@ -39,4 +39,4 @@ jobs:
           python setup.py build_cmake --inplace
 
       - name: Run pylint
-        run: python3 -m pylint --disable=R,W flamingpy/*.py flamingpy/benchmarks/*.py flamingpy/examples/*.py tests/unit/*.py
+        run: python3 -m pylint --disable=R,W flamingpy/**.py tests/**.py

```

To recurse into subdirectories you should extend the glob pattern. Currently, if I try with the glob expression here what I get is the following
```bash
>> ls flamingpy/**.py
flamingpy/__init__.py  flamingpy/_version.py  flamingpy/simulations.py
```
Note this is shell dependent but this suggestion is the most general possible expression.

```suggestion
        run: python3 -m pylint --disable=R,W flamingpy/**/*.py tests/**/*.py
```

---

Yeah, you are absolutely right, this is shell-dependent, it worked on my system, but no guarantee for it to work everywhere. I am happy with your suggestion.